src/tmpfiles/tmpfiles.c

```diff
@@ -1134,41 +1186,41 @@ static int parse_attribute_from_arg(Item *item) {
         return 0;
 }
 
-static int path_set_attribute(Item *item, const char *path) {
-        _cleanup_close_ int fd = -1;
-        struct stat st;
+static int fd_set_attribute(Item *item, int fd, const struct stat *st) {
+        char procfs_path[strlen("/proc/self/fd/") + DECIMAL_STR_MAX(int)];
+        _cleanup_close_ int procfs_fd = -1;
+        _cleanup_free_ char *path = NULL;
         unsigned f;
-        int r;
+         int r;
```

Comment:
Something is off with indentation here. Starting from this line until line 1209, indentation is 9 spaces, not 8.